recipes/calculix/bld.bat

```diff
@@ -0,0 +1,14 @@
+cd ccx*/src
+
+rm Makefile_MT
+CP %RECIPE_DIR%\Makefile_MT Makefile_MT
+CP %BUILD_PREFIX%\Library\mingw-w64\bin\mingw32-make %BUILD_PREFIX%\Library\mingw-w64\bin\make
```

Comment:
is there a better way to get a make-command? I guess the source-code should use $(MAKE) instead of make.